src/drive/web/modules/services/components/SuggestionProvider.jsx

```diff
@@ -46,21 +48,37 @@ class SuggestionProvider extends React.Component {
     )
   }
 
-  // fetches pretty much all the files and preloads FuzzyPathSearch
+  // fetches pretty much all the files not trashed and preloads FuzzyPathSearch
   async indexFiles() {
     const { client } = this.props
     // TODO: fix me
     // eslint-disable-next-line no-async-promise-executor
     return new Promise(async resolve => {
-      const resp = await cozy.client.fetchJSON(
-        'GET',
-        `/data/io.cozy.files/_all_docs?include_docs=true`
+      const files = await client.queryAll(
+        Q(DOCTYPE_FILES)
+          .partialIndex({
+            _id: {
+              $ne: TRASH_DIR_ID
+            },
+            trashed: {
+              $or: [
+                {
+                  $exists: false
+                },
+                {
+                  $eq: false
+                }
+              ]
+            }
+          })
+          .select(['_id', 'trashed', 'dir_id', 'name', 'path'])
+          .where({
+            _id: {
+              $gt: null
+            }
+          })
```

Comment:
can you link this to https://github.com/cozy/cozy-client/issues/1216 ?

Reply:
done, in PR description, inside first commit description and third commit description. Thanks.